src/policy-server/cc_client/client_test.go

```diff
@@ -422,8 +424,22 @@ var _ = Describe("Client", func() {
 
 	Describe("GetSubjectSpaces", func() {
 		BeforeEach(func() {
+			responses := []string{
+				fixtures.SubjectSpacesPage1,
+				fixtures.SubjectSpacesPage2,
+				fixtures.SubjectSpacesPage3,
+			}
 			fakeJSONClient.DoStub = func(method, route string, reqData, respData interface{}, token string) error {
-				_ = json.Unmarshal([]byte(fixtures.SubjectSpaces), respData)
+				parsedRoute, err := url.Parse(route)
+				Expect(err).NotTo(HaveOccurred())
+				pageParameter := parsedRoute.Query().Get("page")
+				if pageParameter == "" {
+					pageParameter = "1"
+				}
+				page, err := strconv.Atoi(pageParameter)
+				Expect(err).NotTo(HaveOccurred())
+				response := responses[page - 1]
+				_ = json.Unmarshal([]byte(response), respData)
```

Comment:
I know it's just a fixture, but can we check the error here (e.g. in case someone tries to update the fixture and messes it up)?